requirments.txt

```diff
@@ -44,5 +44,4 @@ urllib3==1.7.1
 visitor==0.1.3
 wheel==0.24.0
 zipp==3.2.0
-pytest-pylint==0.16.1
```

Comment:
If you're modifying this file, you probably want to remove line 17:
command-not-found==0.3